code/modules/mob/living/simple_animal/friendly/cockroach.dm

```diff
@@ -31,8 +31,8 @@
 	..()
 
 /mob/living/simple_animal/cockroach/proc/squish()
-	adjustBruteLoss(1) // kills a normal cockroach
 	new /obj/effect/decal/cleanable/deadcockroach(get_turf(src))
```

Comment:
Come on, make it only gib when they actually die, this is lazy work and only I get to do that